cg/cli/workflow/balsamic/base.py

```diff
@@ -161,11 +166,13 @@ def store_housekeeper(context: CGConfig, case_id: str, force: bool):
 
     try:
         analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
-        analysis_api.verify_case_config_file_exists(case_id=case_id)
+        analysis_api.verify_case_config_file_exists(case_id=case_id, dry_run=dry_run)
         analysis_api.verify_deliverables_file_exists(case_id=case_id)
-        analysis_api.upload_bundle_housekeeper(case_id=case_id, force=force)
-        analysis_api.upload_bundle_statusdb(case_id=case_id)
-        analysis_api.set_statusdb_action(case_id=case_id, action=None)
+        analysis_api.upload_bundle_housekeeper(case_id=case_id, dry_run=dry_run, force=force)
+        analysis_api.upload_bundle_statusdb(
+            case_id=case_id, comment=comment, dry_run=dry_run, force=force
+        )
+        analysis_api.set_statusdb_action(case_id=case_id, action=None, dry_run=dry_run)
```

Comment:
Probably out of scope for this PR but it would be nice to not have all this logic in the CLI file.

Reply:
Yes I totally agree, but I'm not sure it's worth the effort giving that we'll try to deprecate Balsamic in the next 6-12 months 😓